pageserver/src/basebackup.rs

```diff
@@ -39,16 +38,87 @@ use postgres_ffi::PG_TLI;
 use postgres_ffi::{BLCKSZ, RELSEG_SIZE, WAL_SEGMENT_SIZE};
 use utils::lsn::Lsn;
 
+pub async fn send_basebackup_tarball<'a, W>(
```

Comment:
I think the doc comment 

```
/// Create basebackup with non-rel data in it.
/// ...
```

should be on this function, as it's the public interface.